cmd/root.go

```diff
@@ -120,6 +120,7 @@ func setGlobalOptionsForRootCmd(fs *pflag.FlagSet, globalOptions *config.GlobalO
 	fs.StringArrayVar(&globalOptions.StateValuesSet, "state-values-set", nil, "set state values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2). Used to override .Values within the helmfile template (not values template).")
 	fs.StringArrayVar(&globalOptions.StateValuesFile, "state-values-file", nil, "specify state values in a YAML file. Used to override .Values within the helmfile template (not values template).")
 	fs.BoolVar(&globalOptions.SkipDeps, "skip-deps", false, `skip running "helm repo update" and "helm dependency build"`)
+	fs.BoolVar(&globalOptions.StripArgsValuesOnExitError, "strip-args-values-on-exit-error", false, `On exit error, strip the values of the args`)
```

Comment:
Awesome work @jkroepke!

One suggestion here-

Can we perhaps make this `true` by default following the secure-by-default pattern?
Would it break anything? 🤔 

WDYT @yxxhero?

Reply:
```suggestion
	fs.BoolVar(&globalOptions.StripArgsValuesOnExitError, "strip-args-values-on-exit-error", false, `Strip the potential secret values of the helm command args contained in a helmfile error message`)
```

Maybe this might better communicate that it is a security feature?

Reply:
make sense. @mumoshu